recipes/pydicom/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: pydicom
+  version: "0.9.9"
```

Comment:
@jakirkham made a comment about pulling this out into a single variable (https://github.com/conda-forge/staged-recipes/pull/284/files#r58764815). That is maybe something to consider next time you update this recipe.